webpack/strategies/docs.js

```diff
@@ -23,10 +23,10 @@ export default (config, options) => {
         noParse: /babel-core\/browser/,
         loaders: config.module.loaders
           .map(value => {
-            if (/\.js\/$/.test(value.test.toString())) {
+            if (/js/.test(value.test.toString())) {
```

Comment:
Matt have simplified it in the new `strategies/docs-development.js`
https://github.com/AlexKVal/react-bootstrap/blame/master/webpack/strategies/docs-development.js#L20
`strategies/docs-development.js` has been copied from this `strategies/docs.js`

This line filters not files and rules instead.
Rules from here https://github.com/AlexKVal/react-bootstrap/blob/master/webpack/webpack.config.js#L47-L49